### DISCUSSION TITLE
plugin gets stuck in loading mode

### DISCUSSION THREAD

**Discussion by morulaus** [YukiGasai/obsidian-google-calendar#82] [Q&A] [answered]
Hello,

thank you for a wonderful plugin! Unfortunately after a few days of working great it now gets stuck in LOADING mode. Auth seems to be working as I'm able to open a web view from the command menu. Here's the code I'm using to see one day schedule that gets stuck in LOADING:

```gEvent
type: schedule
date: today
timespan: 0
```

**Comment by YukiGasai** [comment]
Hi, please try to log out and login again.
If this doesn't help, are you using your own client?

**Reply by freqyfreqy** [reply]
Hi Yukigasai. Thank you so much. I have the exact same problem. 

I am using my own client. 

This works and I get the webview
        type: web

This worked before and after a few days doesn't work and I get the 'Loading' sign. 
        type: schedule
<img width="389" alt="image" src="https://user-images.githubusercontent.com/96843562/226985736-7a62e781-00e9-4b8f-8e06-052a6d9bca10.png">


I'm not sure logging off and logging in will work because the webview works. I'll try and report back to you.

**Reply by freqyfreqy** [reply]
I logged out, restarted obsidian, and logged back in and now it seems to work. I'll monitor over the next few days. BTW, I also use the mobile app so I have to use the Google Calendar Refresh token. When I logged out from my desktop, I had to logout and reenter the token on my mobile. Now both seem to work. I'll let you know if it stops working again. Thanks

**Reply by freqyfreqy** [reply]
I keep having this problem. I updated the plugin and now have this issue again. I uninstalled the plugin and reinstalled and relogged into google. 
type: web -> works
type: schedule -> shows loading

Is there a debug mode which can help me find out more informaiton? Perhaps I'm being rate limited because i think I left the settings to update once 10 seconds. Not sure if this is the problem

**Reply by freqyfreqy** [reply]
I believe my issue now is related to this bug I believe I discovered: https://github.com/YukiGasai/obsidian-google-calendar/issues/106

**Reply by YukiGasai** [reply]
I found a problem that could have caused this as well, should get better with the new update.

**Reply by freqyfreqy** [reply, marked as answer]
thanks！
I think my problems was not using [ ] brackets!
